command/agent/consul/script_test.go

```diff
@@ -0,0 +1,173 @@
+package consul
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/nomad/helper/testtask"
+	"github.com/hashicorp/nomad/nomad/structs"
+)
+
+func TestMain(m *testing.M) {
+	if !testtask.Run() {
+		os.Exit(m.Run())
+	}
+}
+
```

Comment:
Added except "Unknown command" which depends on the Exec implementation to return an error (which is tested): https://github.com/hashicorp/nomad/pull/2467/commits/9c14e8be8cec0fe70262472bd2274303ae04b830

Exec implementations are tested directly for example: https://github.com/hashicorp/nomad/pull/2467/files#diff-1eb2911dd4ed9b612194897284c68251R304